### PR TITLE
Update pdfjs to v2.9

### PR DESCRIPTION
**Pull request by JanAckermann**
### Description
* Update pdfjs from v2.5 to v2.9 (which enabled signature support)
* Drop support for IE and non-chromium-based EDGE due to pdfjs cancel support since v2.7 
  * Download will be started instead of opening with files_pdfviewer while click on a pdf file
  * Context menu item 'Open in PDFViewer' won't be displayed anymore 

### Fixes
https://github.com/owncloud/files_pdfviewer/issues/286 
https://github.com/owncloud/files_pdfviewer/issues/258
https://github.com/owncloud/enterprise/issues/4680

Might fix:
https://github.com/owncloud/files_pdfviewer/issues/275

### Screenshots

Before:
image

After:
image

### CODE REVIEW

File: js/workersrc.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ function redirectIfNotDisplayedInFrame () {
 redirectIfNotDisplayedInFrame();
 
 function deferredViewerConfig() {
+	console.log(pdfjsLib);
```
Comment (VicDeo) [inline js/workersrc.js:17]: debug leftover?

Comment (JanAckermann) [reply js/workersrc.js:17]: @VicDeo yes, far away from ready to review at this point

**Comment by JanAckermann** [comment]
Ready to review

**Comment by jvillafanez** [comment]
I'm not sure if falling back to download the file is a good idea. It might be better to show a "your browser isn't supported" notification. The problem I see is that the behaviour is very different: I'm on my phone / tablet, touch the file and it opens the pdf, however when I'm on the office using IE 11 (company policy) and I click the file, it downloads instead of opening. It seems like a bug to me.
At least, with the notification, the user knows what happens and can choose between downloading the file or use a different browser to open the file.

**Comment by JanAckermann** [comment]
@jvillafanez this has been discussed with @hodyroff and @tbsbdr as files_mediaviewer does the same, the decision was made.

**Comment by jvillafanez** [comment]
I still think there will be bug reports incoming complaining about the behaviour, but if it was decided... :man_shrugging:

**Review by jvillafanez — APPROVED** [review]

**Pull request merged**